wsd/DocumentBroker.cpp

```diff
@@ -336,7 +342,7 @@ void DocumentBroker::pollThread()
         // ignore load time out
         std::getenv("PAUSEFORDEBUGGER") ? -1 :
 #endif
-            ConfigUtil::getConfigValue<int>("per_document.limit_load_secs", 100);
+            getLimitLoadSecs();;
```

Comment:
Nit: extra semi-colon.

Reply:
merge conflict needs a rebase anyway, so I'll fix this unaccountable ;;